tests/appsec/blocking_rule.json

```diff
@@ -575,6 +575,76 @@
         "block"
       ]
     },
+    {
+      "id": "tst-037-013",
+      "name": "Test block on multiple request addresses (without path_params)",
+      "tags": {
+        "type": "lfi",
+        "crs_id": "000013",
+        "category": "attack_attempt"
+      },
+      "conditions": [
+        {
+          "parameters": {
+            "inputs": [
+              {
+                "address": "server.request.method"
+              }
+            ],
+            "regex": "GET"
+          },
+          "operator": "match_regex"
+        },
+        {
+          "parameters": {
+            "inputs": [
+              {
+                "address": "server.request.uri.raw"
+              }
+            ],
+            "regex": "wX1GdUiWdVdoklf0pYBi5kQApO9i77tN"
```

Comment:
To be honest, I think cryptic values make it hard to read the tests itself. Instead I would like to see something, which will immediately give me a sign in the tests, that I see a value, which suppose to trigger something.

Ideas:

1. malicious-uri
2. security-violation-uri
3. is-attack-uri
4. ...

I know that naming is hard, but you will see how tests change after that

Reply:
Alright, then we can add the "salt" to the good-meaning value, like

```
malicious-uri-wX1GdUiWdVdoklf0pYBi5kQApO9i77tN
```